politeiawww/piwww.go

```diff
@@ -1481,6 +1481,86 @@ func (p *politeiawww) handleCommentNew(w http.ResponseWriter, r *http.Request) {
 	util.RespondWithJSON(w, http.StatusOK, cnr)
 }
 
+func convertVoteFromPi(v pi.CommentVoteT) piplugin.VoteT {
+	switch v {
+	case pi.CommentVoteInvalid:
+		return piplugin.VoteInvalid
+	case pi.CommentVoteDownvote:
+		return piplugin.VoteDownvote
+	case pi.CommentVoteUpvote:
+		return piplugin.VoteUpvote
+	default:
+		return piplugin.VoteInvalid
+	}
+}
+
+func (p *politeiawww) processCommentVote(cv pi.CommentVote, usr user.User) (*pi.CommentVoteReply, error) {
+	log.Tracef("processCommentVote")
```

Comment:
For these trace statements, its good to give info that'd you'd need during debugging since that's really the only time you'd turn tracing on.  I'd go with something like:
```suggestion
	log.Tracef("processCommentVote: %v %v %v", cv.Token, cv.CommentID, cv.Vote)
```